test/test_build.py

```diff
@@ -180,13 +217,20 @@ def test_image_is_generated(image_type):
 
 @pytest.mark.skipif(platform.system() != "Linux", reason="boot test only runs on linux right now")
 @pytest.mark.parametrize("image_type", gen_testcases("direct-boot"), indirect=["image_type"])
-def test_image_boots(image_type):
+def test_image_boots(image_type, build_ansible_container):
```

Comment:
Tests that use the image_type fixture use cached images. I would prefer to just add a new `test_ansible` or similar and not overload the existing test (removing things from it is fine though of course).